src/Main.elm

```diff
@@ -632,17 +671,130 @@ modalView : Model -> Html Msg
 modalView model =
     div maskStyle
         [ modalContent []
-            [ windowsChrome [ style "padding" "0 18px 18px" ]
+            [ windowsChrome [ style "padding" "0 18px 90px" ]
                 [ formGroup "Difficulty"
                     [ radiobutton "Beginner" Beginner model.game
                     , radiobutton "Intermediate" Intermediate model.game
                     , radiobutton "Expert" Expert model.game
+                    , customRadioButton model.menu
                     ]
                 ]
             ]
         ]
 
 
+testDisplay : Maybe Menu -> Html Msg
+testDisplay menu =
```

Comment:
is this used anywhere?

Reply:
oops - was some debugging code i added but forgot to remove.